src/ubuntu-generic-diskimages/kernel-and-modules/arm-ubuntu-22.04/copy_modules_to_host.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Copyright (c) 2025 The Regents of the University of California.
+# SPDX-License-Identifier: BSD 3-Clause
+
+DOCKERFILE="Dockerfile"
+OUTPUT="my-arm-5.15.168-kernel"
+
+# Build the Docker image
+DOCKER_BUILDKIT=1 docker build --no-cache \
+    --file "$DOCKERFILE" \
+    --output "$OUTPUT" .
+
+echo "Build completed for $1: Output directory is $OUTPUT"
```

Comment:
```suggestion
if [ $? -eq 0 ];
then
    echo "Build completed for Arm 24.04 kernel: Output directory is $OUTPUT"
else
    echo "Build failed for Arm 24.04 kernel"
fi
```
This message could be modified to print different messages based on build failure/success. I haven't tested this code snippet, so please test it if you want to incorporate it into the script

Reply:
Could you swap out this line for the following code snippet? This will print different messages on successful/failed kernel build.
```bash
if [ $? -eq 0 ];
then
    echo "Build completed for Arm 22.04 kernel: Output directory is $OUTPUT"
else
    echo "Build failed for Arm 22.04 kernel"
fi
```